integration/package-operator/objectset_test.go

```diff
@@ -418,6 +418,15 @@ func defaultObjectSetRev2(cm1, cm3 *corev1.ConfigMap, rev1 *corev1alpha1.ObjectS
 	}, nil
 }
 
+func isControllerOf(objectReferences []corev1alpha1.ControlledObjectReference, expected corev1alpha1.ControlledObjectReference) bool {
+	for _, ref := range objectReferences {
+		if reflect.DeepEqual(ref, expected) {
```

Comment:
Isn't there a nicer way than using reflect here? :D I mean, in k8s it seems to be usual but I can't help it